src/v3/preact.config.js

```diff
@@ -105,7 +100,8 @@ export default {
           ...rule,
           use: rule.use.reduce((acc, loader) => {
             acc.push(
-              loader !== MiniCssExtractPlugin.loader ? loader : 'style-loader',
+              // force MiniCssExtractPlugin.loader for watch mode and static builds
+              /style-loader/.test(loader) ? MiniCssExtractPlugin.loader : loader,
```

Comment:
doesn't look like this is matching anything right now:
```js
      if (rule.use) {
        return {
          ...rule,
          use: rule.use.reduce((acc, loader) => {
            console.log(/style-loader/.test(loader) && loader); // false for everything on `yarn build`
            acc.push(
              // force MiniCssExtractPlugin.loader for watch mode and static builds
              /style-loader/.test(loader) ? MiniCssExtractPlugin.loader : loader,
            );
            return acc;
          }, []),
        };
      }
```

Reply:
This only matches when you run `dev` - the "watch mode" causes it to run `style-loader` rather than `MiniCssExtractPlugin.loader`